test_regress/t/t_pipe_filter.pl

```diff
@@ -13,7 +13,7 @@
 my $stdout_filename = "$Self->{obj_dir}/$Self->{name}__test.vpp";
 
 compile(
-    verilator_flags2 => ['-E --pipe-filter \'python3 t/t_pipe_filter.pf\' '],
+    verilator_flags2 => ['--no-std', '-E --pipe-filter \'python3 t/t_pipe_filter.pf\' '],
```

Comment:
Please expand on why all these tests are changed. I know you mentioned some in the description, but the reviewer should know what's happening in each one (so just describe each category of test changed).

Reply:
I've modified PR description + added a detailed list of tests grouped by the reason of their modification.